lib/string_decoder.js

```diff
@@ -58,8 +58,8 @@ const StringDecoder = exports.StringDecoder = function(encoding) {
 // returned when calling write again with the remaining bytes.
 //
 // Note: Converting a Buffer containing an orphan surrogate to a String
-// currently works, but converting a String to a Buffer (via `new Buffer`, or
-// Buffer#write) will replace incomplete surrogates with the unicode
+// currently works, but converting a String to a Buffer (via `Buffer.from()`,
```

Comment:
Should it be better that we keep the same writing style as afterwards `Buffer#write` and use the `Buffer#from` here ?